app/frontend/src/routes.ts

```diff
@@ -32,8 +32,7 @@ export const logoutRoute = "/logout";
 export const connectionsRoute = "/connections";
 export const friendsRoute = `${connectionsRoute}/friends`;
 
-export const userRoute = "/user";
-export const routeToUser = (username: string) => `${profileRoute}/${username}`;
+export const routeToUser = (username: string) => `${userRoute}/${username}`;
```

Comment:
Just a small nitpick - might make more sense to move this line up with all of the other user routes so they are grouped together in this file.